### PR TITLE
Store paths to additional files per Execution in the DB 

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Add new column in Execution table: additional_files
* Store paths to additional files per Execution in the DB
* Update tests

### CODE REVIEW

File: save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt [hunk 1]
```diff
@@ -132,9 +135,22 @@ class CloneRepositoryController(
...
-    private fun Flux<FileInfo>.collectToMultipart(multipartBodyBuilder: MultipartBodyBuilder) = map {
-        multipartBodyBuilder.part("file", fileSystemRepository.getFile(it))
+    @Suppress("TYPE_ALIAS")
+    private fun Flux<FileInfo>.collectToMultipartAndUpdateExecution(
+        multipartBodyBuilder: MultipartBodyBuilder,
+        execution: Execution
+    ): Mono<List<MultipartBodyBuilder.PartBuilder>> {
+        val additionalFiles = StringBuilder("")
+        return map {
+            val path = Paths.get(it.uploadedMillis.toString()).resolve(it.name)
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt:145]: You should use logic from `fileSystemRepository` to calculate the path

**Review by petertrr — CHANGES_REQUESTED** [review]

File: save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt]: Used `getPath` method from `fileSystemRepository`

File: save-cloud-common/src/commonMain/kotlin/org/cqfn/save/execution/ExecutionDto.kt [hunk 2]
```diff
@@ -23,4 +24,5 @@ data class ExecutionDto(
     val passedTests: Long,
     val failedTests: Long,
     val skippedTests: Long,
+    val additionalFiles: String?,
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-cloud-common/src/commonMain/kotlin/org/cqfn/save/execution/ExecutionDto.kt]: Changed

File: db/changelog/v-2/05-changeset-execution.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="5" author="kgevorkyan" context="dev or prod">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="execution"/>
+        </preConditions>
+        <addColumn tableName="execution">
+            <column name="additional_files" type="varchar(8000)" defaultValue="null"/>
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply db/changelog/v-2/05-changeset-execution.xml]: Fixed

**Review by petertrr — APPROVED** [review]

**Pull request merged**